### PR TITLE
Adds toleration support to KH checks.

### PR DESCRIPTION
**Pull request by zjhans**

### CODE REVIEW

File: .ci/values.yaml [hunk 1]
```diff
@@ -65,6 +70,11 @@ check:
       tag: v3.2.1
     extraEnvs:
     nodeSelector: {}
+    tolerations:
```
Comment (joshulyne) [inline .ci/values.yaml:73]: maybe put `tolerations: []` so we know it takes in a list?

**Review by joshulyne — APPROVED** [review]
other than the nitpicky comment, lgtm!

File: .ci/values.yaml (hunk 1 above)
Comment (zjhans) [reply .ci/values.yaml]: Updated, now with brackets! 😃

**Comment by integrii** [comment]
As soon as the comments in the `.ci/values.yaml` are pruned, this looks good to go!

**Review by joshulyne — APPROVED** [review]

**Pull request merged**